calico_containers/integrations/kubernetes/calico_kubernetes.py

```diff
@@ -326,9 +339,18 @@ def _apply_tags(self, profile_name, pod):
             print('No labels found in pod %s' % pod)
             return
 
+        # Grab namespace and create a tag if it exists.
+        namespace = self._get_namespace(pod)
+        if namespace:
+            tag = self._label_to_tag('namespace', namespace, None)
```

Comment:
I'd define 'namespace' as NAMESPACE_PREFIX up at the top.  Will make this code easier to understand.